shared/menubar/index.render.desktop.js

```diff
@@ -409,6 +410,13 @@ const styles = {
     position: 'absolute',
     right: 0,
     top: 5
+  },
+  dz1InputHint: {
+    color: globalColors.black,
+    // This is because normal <Input/> does center-aligned hints by using
+    // width: 100% on position: absolute
+    width: '',
```

Comment:
Yeah, it works.  I've changed it to "width: 'auto'" now, that works too and makes it clearer, thanks.